javascript/core/functions-i/function-arguments.md

```diff
@@ -32,27 +26,28 @@ aspects:
 ---
 ## Content
 
-When we pass primitive variables (such as numbers and strings) to functions in JavaScript, they are *passed by value* which means that if we manipulate the argument value inside the function, changes will not be reflected outside the function.
-
-However, if we were to pass an object then changes *are* reflected on the object itself.
+When we pass primitive variables (such as numbers and strings) to functions in JavaScript, they are *passed by value*. This means that if we manipulate the argument value inside the function, changes will not be reflected outside the function.
 
-For example:
-```
-function changeVal(x){
-    x=1000;
+```js
+function changeVal(x) {
+  x = 1000;
 }
 
 var x = 5;
 changeVal(x);
-console.log(x);//still 5
+console.log(x); // still 5
+```
 
-function changeObj(x){
-    x.name="Apple";
+However, if we were to use an object as argument, then it will **not be pased as a value**. What will happen though is that the object will **be passed as a reference**, which means that all the changes on the object made inside the function **are** reflected on the object itself.
```

Comment:
In pair with the comment above:

```suggestion
When we pass non-primitive values (such as objects and arrays) into a function, we are not passing in a copy and instead are passing in the actual value. This behavior is called **pass by reference**. If we were to manipulate one of those values, we will also end up changing original value outside of the function. 
```

Reply:
As far as clarifying what a "reference" is, I wonder if we should do it right here?

```suggestion
When we pass non-primitive values (such as objects and arrays) into a function, we are not passing in a copy and instead are passing in the actual value. This behavior is called **pass by reference**, where the "reference" means "direct link to the value". If we were to manipulate one of those values, we will also end up changing original value outside of the function. 
```

Reply:
I like this one more. It's just an extra phrase but might help users in the long run.

Reply:
@kapnobatai137 coolio